usaspending_api/awards/v2/views/idvs/funding_treemap.py

```diff
@@ -0,0 +1,96 @@
+from collections import OrderedDict
+from copy import deepcopy
+
+from psycopg2.sql import Identifier, Literal, SQL
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.common.helpers.sql_helpers import execute_sql_to_ordered_dictionary
+from usaspending_api.common.views import APIDocumentationView
+from usaspending_api.common.validator.award import get_internal_or_generated_award_id_model
+from usaspending_api.common.validator.tinyshield import TinyShield
+
+FUNDING_TREEMAP_SQL = SQL("""
+    with cte as (
+        select    award_id
+        from      parent_award
+        where     {award_id_column} = {award_id}
+        union all
+        select    cpa.award_id
+        from      parent_award ppa
+                  inner join parent_award cpa on cpa.parent_award_id = ppa.award_id
+        where     ppa.{award_id_column} = {award_id}
+    )
+    select
+        {columns}
+    from
+        cte
+        inner join awards pa on
+            pa.id = cte.award_id
+        inner join awards ca on
+            ca.parent_award_piid = pa.piid and
+            ca.fpds_parent_agency_id = pa.fpds_agency_id and
+            ca.type not like 'IDV\_%'
```

Comment:
No need for `\_`, simply using `IDV%` is preferable.

Reply:
While it's not necessary, this is pretty much how I've been doing it in all my work.  Unless there's a specific reason not to do it this way (like it's wrong, for example), I'd rather we be consistent.

On the other hand, if we feel really strongly about changing this one, let's change them all.

Reply:
The others have already been changed.

Reply:
Well ok then... I drop my objection!